docs/release_notes/v1.9.2.md

```diff
@@ -0,0 +1,22 @@
+# Dapr 1.9.2
+
+### Fixes panics when using pubsub subscriptions or input bindings via gRPC with tracing disabled
+
+#### Problem
+
+Users who disabled tracing and are subscribing to Dapr pubsub components via gRPC or using input bindings via gRPC will encounter panics when an event is attempted to be delivered.
+
+### Impact
+
+This issue impacts users who:
+
+- Disabled tracing (sampling rate is set to 0, not the default Dapr config sets this to 1)
+- Use input bindings via gRPC and/or subscribe to pubsub components via gRPC
+
+#### Root cause
+
+A code refactor of a tracing utility function previously required by changes in our upstream tracing libraries changed the return signature from a pointer to a value. Several `nil` checks were missing in the code resulting in panics. This previously presented no concern because a method call on a nil pointer is a `no-op` in Go.
```

Comment:
I think this is a bit too detailed for the average reader who isn't aware of Dapr's internals.

How about: `Dapr 1.9.0 added support for tracing with OpenTelemetry, which uses a new SDK version. During the upgrade, an error was introduced causing a panic when tracing was disabled, due to a missing "nil-check".`

Reply:
Please use the suggestion method

Reply:
```suggestion
Dapr 1.9.0 added support for tracing with OpenTelemetry, which uses a new SDK version. During the upgrade, an error was introduced causing a panic when tracing was disabled, due to a missing "nil-check".
```